package connect4.exceptions;

public class FinishedGameException extends Exception {
}
